app/Actions/ConfigureFrontendFramework.php

```diff
@@ -2,36 +2,77 @@
 
 namespace App\Actions;
 
-use App\Shell\Shell;
+use App\Shell;
+use Illuminate\Support\Facades\Config;
 
 class ConfigureFrontendFramework
 {
     use LamboAction;
 
     private $shell;
-
     private $laravelUi;
+    private $availableFrontends;
 
     public function __construct(Shell $shell, LaravelUi $laravelUi)
     {
         $this->shell = $shell;
         $this->laravelUi = $laravelUi;
     }
 
-    public function __invoke()
+    public function __invoke(array $availableFrontends)
     {
-        if (! config('lambo.store.frontend')) {
+        $this->availableFrontends = $availableFrontends;
+
+        if (! Config::get('lambo.store.frontend')) {
             return;
         }
 
-        $this->laravelUi->install();
+        app('console-writer')->logStep('Configuring frontend scaffolding');
+
+        if ($this->continueInstallation()) {
+            $this->laravelUi->install();
```

Comment:
Can we do two things?

1. Rename `install()` to `ensureInstalled()` to make it a bit clearer?

2. Add a method named something like `installFrontend($frontend)`?
```php
$this->laravelUi->installFrontend(config('lambo.store.frontend'));
```

3. In the `installFrontend` method, if it fails, throw an exception with this message:

```php
throw new LamboException(sprintf("Installation of %s UI scaffolding did not complete successfully.", config('lambo.store.frontend')));
```

Reply:
Parking for now since the release of Laravel Jetstream will change how we do frontend scaffolding.